src/features/views/components/ViewDataTable/ViewDataTableToolbar.tsx

```diff
@@ -55,16 +57,23 @@ const ViewDataTableToolbar: React.FunctionComponent<
   };
   return (
     <Box role="toolbar">
-      <Slide direction="left" in={!!selection.length} timeout={150}>
-        <Button
-          data-testid="ViewDataTableToolbar-createFromSelection"
-          disabled={disabled}
-          onClick={onViewCreate}
-          startIcon={<Launch />}
-        >
-          <Msg id={messageIds.toolbar.createFromSelection} />
+      {!isLoading && (
+        <Slide direction="left" in={!!selection.length} timeout={150}>
+          <Button
+            data-testid="ViewDataTableToolbar-createFromSelection"
+            disabled={disabled}
+            onClick={onViewCreate}
+            startIcon={<Launch />}
```

Comment:
How about something like this instead?

```suggestion
            disabled={disabled || isLoading}
            onClick={onViewCreate}
            startIcon={isLoading? <CircularProgress size={25}> : <Launch />}
```